Packs/FeedGithub/Integrations/FeedGithub/FeedGitHub_description.md

```diff
@@ -0,0 +1,3 @@
+## Feed Hello World
+- This text is markdown
```

Comment:
here we should put a description for how to set up the integration. See what is done to othe github integrations